src/envo/envo-edit.owl

```diff
@@ -42215,6 +42226,7 @@ ObjectPropertyAssertion(<http://purl.obolibrary.org/obo/BFO_0000050> <https://ww
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_53243> "envoPlastics"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_53550> "envoPlastics"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_60034> "envoPlastics"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_60160> "envoPlastics")
```

Comment:
I note this doesn't have a language tag - intentional? As @cmungall noted in our Slack - these should actually be entered via the IRI editor, rather than literals anyway

Reply:
> I note this doesn't have a language tag - intentional?
I think this is due different people adding the subset tag. 

> As @cmungall noted in our Slack - these should actually be entered via the IRI editor, rather than literals anyway
I'll open a separate issue on this.